### DISCUSSION TITLE
Should we update the docs to update references to https://github.com/vmware-research

### DISCUSSION THREAD

**Discussion by gapisback** [ccc-certifier-framework/certifier-framework-for-confidential-computing#238] [General]
Now, that the certifier Framework repo has been moved to under `github.com:ccc-certifier-framework`, seems like the references to `vmware-research` in a few places in the docs should be updated to refer to the `ccc-certifier-framework` location.

Here is a short-list:

```
/Users/aditya/Personal/Projects/Code/certifier-framework-for-confidential-computing

Macintosh:[259] $
Macintosh:[259] $ grep vmware-research `find . -name "*.md" -print`

./CONTRIBUTING.md:git remote add upstream https://github.com/vmware-research/certifier-framework-for-confidential-computing.git
./CONTRIBUTING.md:When your branch gets out of sync with the vmware-research/main branch, use the following to update:
./sample_apps/simple_app_under_gramine/instructions.md:git clone https://github.com/vmware-research/certifier-framework-for-confidential-computing.git
./Doc/install-certifier-Ubuntu-20.04.md:git clone https://github.com/vmware-research/certifier-framework-for-confidential-computing.git
./src/gramine/gramine_tests/instructions.md:git clone https://github.com/vmware-research/certifier-framework-for-confidential-computing.git certifier
./deprecated/simple_app_under_asylo_secure_grpc/README.md:https://github.com/vmware-research/certifier-framework-for-confidential-computing.git
```

@jlmucb -- if you agree, I'd be happy to open a docs-update issue and do this cleanup.

**Comment by yelvmw** [comment]
I remember the VMware-research links should be redirected automatically already? Though fixing them makes sense too.

Ye


> On Apr 1, 2024, at 3:27 PM, Aditya P. Gurajada ***@***.***> wrote:
> 
> 
> Now, that the certifier Framework repo has been moved to under github.com:ccc-certifier-framework, seems like the references to vmware-research in a few places in the docs should be updated to refer to the ccc-certifier-framework location.
> 
> Here is a short-list:
> 
> /Users/aditya/Personal/Projects/Code/certifier-framework-for-confidential-computing
> 
> Macintosh:[259] $
> Macintosh:[259] $ grep vmware-research `find . -name "*.md" -print`
> 
> ./CONTRIBUTING.md:git remote add upstream https://github.com/vmware-research/certifier-framework-for-confidential-computing.git
> ./CONTRIBUTING.md:When your branch gets out of sync with the vmware-research/main branch, use the following to update:
> ./sample_apps/simple_app_under_gramine/instructions.md:git clone https://github.com/vmware-research/certifier-framework-for-confidential-computing.git
> ./Doc/install-certifier-Ubuntu-20.04.md:git clone https://github.com/vmware-research/certifier-framework-for-confidential-computing.git
> ./src/gramine/gramine_tests/instructions.md:git clone https://github.com/vmware-research/certifier-framework-for-confidential-computing.git certifier
> ./deprecated/simple_app_under_asylo_secure_grpc/README.md:https://github.com/vmware-research/certifier-framework-for-confidential-computing.git
> @jlmucb <https://www.google.com/url?q=https://github.com/jlmucb&source=gmail-imap&ust=1712604433000000&usg=AOvVaw2UpUhxU4kuDWiPpl-rLJj9> -- if you agree, I'd be happy to open a docs-update issue and do this cleanup.
> 
> —
> Reply to this email directly, view it on GitHub <https://www.google.com/url?q=https://github.com/ccc-certifier-framework/certifier-framework-for-confidential-computing/discussions/238&source=gmail-imap&ust=1712604433000000&usg=AOvVaw2h_46zVVVet8xFR7vjLxrz>, or unsubscribe <https://www.google.com/url?q=https://github.com/notifications/unsubscribe-auth/A2FOFZRQOQ4RAAK3R4NMJHLY3GYIXAVCNFSM6AAAAABFSCRRT6VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGQ2DONRRGI&source=gmail-imap&ust=1712604433000000&usg=AOvVaw2cDiMapVVzaJQdLvVpFWb0>.
> You are receiving this because you are subscribed to this thread.
> 


-- 
This electronic communication and the information and any files transmitted 
with it, or attached to it, are confidential and are intended solely for 
the use of the individual or entity to whom it is addressed and may contain 
information that is confidential, legally privileged, protected by privacy 
laws, or otherwise restricted from disclosure to anyone else. If you are 
not the intended recipient or the person responsible for delivering the 
e-mail to the intended recipient, you are hereby notified that any use, 
copying, distributing, dissemination, forwarding, printing, or copying of 
this e-mail is strictly prohibited. If you received this e-mail in error, 
please return the e-mail to the sender, delete it from your computer, and 
destroy any printed copy of it.
